ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -151,7 +151,8 @@ def get_terms_of_use_url(terms_of_use):
 
 
 def get_dataset_terms_of_use(pkg):
-    rights = logic.get_action('ogdch_dataset_terms_of_use')({}, {'id': pkg})
+    rights = logic.get_action('ogdch_dataset_terms_of_use')(
+        {}, {'id': pkg['id']})
```

Comment:
This seems to have been a bug - the helper wasn't in use anywhere so we didn't see it failing.

Reply:
I think this function is not used any more. Can you double check this (I checked already) and remove it if it is unused.

Reply:
It is used now, in this pull request.